pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -221,6 +230,8 @@ public void processSqlWithMultiStageQueryEngineGet(
   })
   @ManualAuthorization
   public void processSqlWithMultiStageQueryEnginePost(String query, @Suspended AsyncResponse asyncResponse,
+      @ApiParam(value = "Return a cursor instead of complete result set") @QueryParam("getCursor") Boolean getCursor,
+      @ApiParam(value = "Number of rows to fetch") @QueryParam("numRows") Integer numRows,
```

Comment:
Does `numRows` affect the number of rows to fetch even if `getCursor` is false? In case it doesn't I think we should specify that in the description

Reply:
No. `numRows` is ignored if `getCursor` is false. I'll add it to the description.

Reply:
Done